### PR TITLE
[INLONG-877][Improve] Update document of TubeMQ extract node

### LINKED ISSUE #877: [Improve] Update document of TubeMQ extract node
### Description

Update document of TubeMQ extract node

### Are you willing to submit PR?

- [X] Yes, I am willing to submit a PR!

### Code of Conduct

- [X] I agree to follow this project's [Code of Conduct](https://www.apache.org/foundation/policies/conduct)

### PR DESCRIPTION
**Pull request by fancycoderzf**
### Prepare a Pull Request

- Fixes #877 

### Motivation

*Update document of TubeMQ extract node*

### Modifications

*Update document of TubeMQ extract node*

According to these lines, `masterRpc` should be `master.rpc` and `groupId` should be `group.id`.
https://github.com/apache/inlong/blob/18c39ff593b1afb71063d85dc89adc4d589516cf/inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/tubemq/src/main/java/org/apache/inlong/sort/tubemq/table/TubeMQOptions.java#L107
https://github.com/apache/inlong/blob/18c39ff593b1afb71063d85dc89adc4d589516cf/inlong-sort/sort-flink/sort-flink-v1.13/sort-connectors/tubemq/src/main/java/org/apache/inlong/sort/tubemq/table/TubeMQOptions.java#L113

### CODE REVIEW

File: docs/data_node/extract_node/tube.md [hunk 1]
```diff
@@ -46,9 +46,9 @@ Flink SQL> CREATE TABLE tube_extract_node (
      ) WITH (
      'connector' = 'tubemq',
      'topic' = 'topicName',
-     'masterRpc' = 'rpcUrl', -- 127.0.0.1:8715
+     'master.rpc' = 'rpcUrl', -- 127.0.0.1:8715
      'format' = 'json',
-     'groupId' = 'groupName');
+     'group.id' = 'groupName');
```
Comment (vernedeng) [inline docs/data_node/extract_node/tube.md:51]: Does group.Id means InlongGroupId？

Comment (fancycoderzf) [reply docs/data_node/extract_node/tube.md:51]: No, It means TubeMQ's consumer group name.

Comment (dockerzhang) [reply docs/data_node/extract_node/tube.md:51]: change to `group.name`?

Comment (fancycoderzf) [reply docs/data_node/extract_node/tube.md]: done, change `group.id` to `group.name`.  The relevant code change is: PR #9031 https://github.com/apache/inlong/pull/9031, Commit https://github.com/apache/inlong/pull/9031/commits/618216091a1281d5617c33a7a9e1553be476bee9

**Comment by dockerzhang** [comment]
group.id changed in https://github.com/apache/inlong/pull/9031

**Review by gosonzhang — APPROVED** [review]

**Review by EMsnap — APPROVED** [review]

**Pull request merged**